HarmonyPatches/ImproveBasegameSearch.cs

```diff
@@ -47,8 +49,8 @@ static class ImproveBasegameSearch {
 
 				new CodeInstruction(OpCodes.Dup),
 				new CodeInstruction(OpCodes.Ldc_I4_2),
-				matcher.NamedMatch("L_previewBeatmapLevel"),
-				new CodeInstruction(OpCodes.Callvirt, AccessTools.PropertyGetter(typeof(IPreviewBeatmapLevel), nameof(IPreviewBeatmapLevel.levelAuthorName))),
+				matcher.NamedMatch("L_beatmapLevel"),
+				Transpilers.EmitDelegate<Func<BeatmapLevel, string>>(level => level.allMappers.Concat(level.allLighters).Distinct().Join(null, " ")),
```

Comment:
Whats the motiviation for adding Lighters too? I dont think anyone would actually know to search for them since they arent usually exposed

Reply:
I simply used @Meivyn 's fix because I did not figure out how to fix this at first.

Reply:
Because that's what everything else uses.

Reply:
I see, I dont really care that much about it tho to incur such an unnecessary overhead tbh, I'll remove lighters

Reply:
Calling this overhead is a bit of an overstatement but well, it's up to you.

Reply:
Its doing thousands of linq concats, thats certainly not zero cost